_minutes/2023-09-12-wecg-tpac.md

```diff
@@ -0,0 +1,242 @@
+ * [rob] I wanted to highlight this as it's a related capability we should support. I raised this in early discussions on the user script API design ([issue 279](https://github.com/w3c/webextensions/issues/279)), as the primitive is relevant to extensions that are not user script managers ([issue 284](https://github.com/w3c/webextensions/issues/284)). I'll flesh out a proposal and submit it for review in the group.
+
+(WebAuth RP ID – Devlin to kick off to others)
+
+ * [Issue 238](https://github.com/w3c/webextensions/issues/238): WebExtensions should use asset links model for WebAuthn RP ID
+ * [rew] I have a slide deck to help. We have a simplified representation of the WebAuthN flow.
+   * Link to slides: “RP ID and web extensions“ https://docs.google.com/presentation/d/1p6uuyS0mjqQbX39X0iny_aXLzto8HZspt1SYlUFng7M/edit
+   * [rob] FYI: Slide deck has already publicly referenced in issue 238 at https://github.com/w3c/webextensions/issues/238#issuecomment-1623965791
+ * [rew] You call navigator.credentials.create. The credential you create will be tied to the origin. In a browser extension, you can do the same, but the “RP ID” will not necessarily be tied to the website. For example, Dashlane may create multiple applications and there is no way to associate the two.
+ * [rew] Today mobile apps can associate themselves with a domain. This links the bundle ID or app ID to the domain, but there's no equivalent in extensions. There's no association between the extension's origin and the application URL.
+ * [devlin] If you install a Chrome and Edge extension the RPID will be based on the origin including the scheme. So even with two extensions with the same ID they will not be compatible.
+ * [rew] One proposal is to introduce an equivalent ability to link an extension origin to a domain. The OS for mobile apps validates this link. I'm not sure what would perform this association.
+ * [tim cappalli] The browser would be the equivalent to the OS in this model.
+ *
+ * [tomislav] Is that the main use case? Let's get to what you're trying to achieve.
+ * [tim cappalli] We're essentially looking to associate a web domain to an extension-specific scheme.
+ * [rew] It could also be used on a web app
+ * [tim cappalli] If I create a web application for the Dashlane mobile app, it's automatically associated with the Dashlane origin.
+ * [devlin] I discussed this a little with Rew earlier and I wasn't sure why Dashlane doesn't just embed an iframe and use the Dashlane origin from there. That might be viewed as a hack or it might be leveraging the platform.
+ * [tim cappalli] That could work, but I would think that is a hack. The user wouldn't see the extension's origin, they'd see the.
+ * [devlin]
```

Comment:
Do we have thoughts here or should we remove this line? I don't remember what was said.

Reply:
I'll leave it in there; it shows that some discussion has taken place, and with the context the discussion can be filed in.